tests/CodeGen-Binding/Xamarin.Android.LibraryProjectZip-LibBinding/java/JavaLib/library/build.gradle.in

```diff
@@ -2,7 +2,6 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "@BUILD_TOOLS_VERSION@"
```

Comment:
If we remove this, can we remove the MSBuild target that moves `build.gradle.in` to `build.gradle`?

I think `@BUILD_TOOLS_VERSION@` is the only replacement.

Reply:
I removed it based on a deprecation warning from gradle itself

Reply:
Yeah, let's clean up the code here then we don't need it: https://github.com/xamarin/xamarin-android/blob/0d8942fb71141f548e6cfe6aba9b4c294d0962c1/tests/CodeGen-Binding/Xamarin.Android.LibraryProjectZip-LibBinding/Xamarin.Android.LibraryProjectZip-LibBinding.targets#L45-L49

Then just make the file `build.gradle`.

Reply:
@grendello I think the current build failure is because this got removed.

If we use the same build tools version our build bootstraps then we don’t have to accept the license...

We’re you getting a warning though?